src/util-classification-config.c

```diff
@@ -637,14 +637,14 @@ FILE *SCClassConfGenerateInValidDummyClassConfigFD02(void)
 FILE *SCClassConfGenerateInValidDummyClassConfigFD03(void)
```

Comment:
This should really be `Invalid`"
```suggestion
FILE *SCClassConfGenerateInvalidDummyClassConfigFD03(void)
```